src/OpenGL.cpp

```diff
@@ -399,8 +399,10 @@ void OGLRender::TexrectDrawer::init()
 	assert(m_textureBoundsLoc >= 0);
 	m_enableAlphaTestLoc = glGetUniformLocation(m_programTex, "uEnableAlphaTest");
 	assert(m_enableAlphaTestLoc >= 0);
+#ifndef GLES2
 	m_depthScaleLoc = glGetUniformLocation(m_programTex, "uDepthScale");
 	assert(m_depthScaleLoc >= 0);
+#endif
 	glUseProgram(0);
```

Comment:
Please move these changes to another commit. Different issues should be fixed by separate commits. Both commits can be in one pull request.